.node-version

```diff
@@ -0,0 +1 @@
+12.16.3
```

Comment:
Looks like an accidental carryover of https://github.com/open-rpc/server-js/pull/328